javascript-carousel/main.js

```diff
@@ -0,0 +1,77 @@
+var position = 0;
+var interval = setInterval(nextSlide, 3000);
+
+function nextSlide() {
+  addPosition();
+  setImg();
+  circleRotation();
+}
+
+var $clickableDots = document.querySelectorAll('.fa-circle');
+$clickableDots[0].addEventListener('click', function () {
+  position = 0;
+  setImg();
+});
+
+$clickableDots[1].addEventListener('click', function () {
+  position = 1;
+  setImg();
+});
+$clickableDots[2].addEventListener('click', function () {
+  position = 2;
+  setImg();
+});
+$clickableDots[3].addEventListener('click', function () {
+  position = 3;
+  setImg();
+});
+$clickableDots[4].addEventListener('click', function () {
+  position = 4;
+  setImg();
+});
```

Comment:
This is really manual. Can you do this with one event listener possibly? You might make use of a `data-` attribute of some sort